static/styles/night_mode.scss

```diff
@@ -668,6 +668,25 @@ on a dark background, and don't change the dark labels dark either. */
             }
         }
     }
+
+    a.message-control-button {
+        &:hover {
+            /*
+            In night mode the hover color for <a> tag was set  hsl(200, 79%, 66%) in body tag
+            hence !important rule is used to overrride that color
+            */
+            color: hsl(236, 33%, 90%) !important;
+        }
+    }
+
+    .message_controls {
+        .reaction_button, .edit_content, .info > i {
+            color: hsl(236, 33%, 90%, 0.4);
```

Comment:
Can we avoid duplicating `hsl(236, 33%, 90%)` by setting an SCSS variable and then just splitting out the opacity piece to its own line, where relevant?

Reply:
the main issue here is separating the opacity is not working
Because initially the opacity was set 0 for the message box icons
If I separate the alpha from hsl and set opacity 0.4 initially, then there will be no point of initial opacity of 0, the icons are gonna display with opacity 0.4 even when the message box has not hovered

Reply:
The opacity problem can be worked around by making the reaction selector more specific:

~~~ diff
     .message_controls {
-        .reaction_button,
+        .reaction_button > i,
         .edit_content,
         .info > i {
-            color: hsl(236, 33%, 90%, 0.4);
+            color: hsl(236, 33%, 90%);
+            opacity: 0.4;
             &:hover {
-                color: hsl(236, 33%, 90%, 1.0);
+                color: hsl(236, 33%, 90%);
+                opacity: 1.0;
             }
         }
     }

Reply:
And then you don't need to repeat the color in that block.

Reply:
And you can use `.edit_content > i` as well, for consistency.